alethzero/plugins/whisper/Whisper.cpp

```diff
@@ -53,6 +55,17 @@ static Public stringToPublic(QString const& _a)
 		return Public();
 }
 
+static shh::Topics stringToTopics(QString _s)
+{
+	return shh::BuildTopic(_s.toStdString());
+
+	//shh::BuildTopic ret;
```

Comment:
Because it was agreed with Alex, that in this version the will be only single-topic messages. Multitopic option might be added in the future, so I left it commented out.